.github/workflows/grade-publish.yml

```
@@ -24,8 +31,14 @@ jobs:
             - name: Build with Gradle
               run: ./gradlew build
 
+            - name: Set Outputs
+              if: ${{ github.event_name == 'push' }}
+              id: outputs
+              run: echo "COMMITHASH=${GITHUB_SHA::10}" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:40: Double quote to prevent globbing and word splitting

(shellcheck)